arch/risc-v/src/common/riscv_signal_handler.S

```diff
@@ -69,7 +71,7 @@ up_signal_handler:
 
   /* Save ra on the stack */
 
-  addi sp, sp, -16
+  addi sp, sp, -STACK_FRAME_SIZE
```

Comment:
should we use STACK_ALIGNMENT here?

Reply:
Yes. The STACK_ALIGNMENT value is used

Reply:
I mean we don't need quote version here.

Reply:
I did that just to keep same style as in dispatch_syscall, but I can change it.

Reply:
The assembly version of STACK_FRAME_SIZE is not quoted

Reply:
Ok, got it.